plugins/outputs/exec/exec_test.go

```diff
@@ -0,0 +1,57 @@
+package exec
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/serializers"
+	"github.com/influxdata/telegraf/testutil"
+)
+
+func TestExec(t *testing.T) {
+	tests := []struct {
+		command []string
+		err     bool
+	}{
+		{
+			command: []string{"tee"},
+			err:     false,
+		},
+		{
+			command: []string{"sleep", "5s"},
+			err:     true,
+		},
+		{
+			command: []string{"/no/exist", "-h"},
+			err:     true,
+		},
+	}
+
```

Comment:
This test appears to run real commands, so it needs to excluded as part of `go test -short`  It also likely doesn't work on Windows or anywhere that tee/sleep are not in the PATH.